src/common/utils/urlBuilders.ts

```diff
@@ -67,6 +67,26 @@ export const buildNewOrganizationUrl = () => {
   return '/admin/organizations/new';
 };
 
+export const buildCalloutUrl = (
+  calloutNameId: string,
+  hubNameId: string | undefined,
```

Comment:
passing this as undefined doesn't make sense - since it's a function it can be called conditionally only when it's defined.